go/bundle/cmd/dump-bundle/main.go

```diff
@@ -91,7 +93,30 @@ func isTextType(mimeType string) bool {
 	return strings.HasPrefix(m, "text/") || m == "application/javascript"
 }
 
+func checkDoesNotHaveIntegrityBlock(path string) error {
```

Comment:
Can you put this check into `ReadBundleFromFile()` so that the bundle file doesn't have to be opened twice?